### DISCUSSION TITLE
2024/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#36] [Announcements]
# 2024/

🪑?

https://www.supergoodcode.com/2024/

**Comment by Permik** [comment]
2024 IS THE YEAR OF BENCH, CONFIRMED. 🎉🎉🎉

**Comment by qwertychouskie** [comment]
Chair?

**Comment by TeoLinuX** [comment]
Is it the year of chairtop on linux?

**Comment by nwnk** [comment]
congratulations and/or condolences

**Comment by MuaDib1988** [comment]
noooooooo!
please don't leave your chair empty! 
we need you !

**Reply by Dark-Show** [reply]
Benchmarks!

**Comment by ms178** [comment]
Hopefully it's not an empty chair for the whole year?
